bin/cli.js

```diff
@@ -195,6 +195,21 @@ function invoke(env) {
         .catch(exit);
     });
 
+  commander
+    .command('migrate:printAll')
```

Comment:
only `print` is enough

```suggestion
    .command('migrate:print')
```